src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IDynamicInvocationExpression.vb

```diff
@@ -535,6 +536,437 @@ IDynamicInvocationOperation (OperationKind.DynamicInvocation, Type: System.Objec
 
             VerifyOperationTreeAndDiagnosticsForTest(Of InvocationExpressionSyntax)(source, expectedOperationTree, expectedDiagnostics)
         End Sub
+
+        <CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)>
+        <Fact()>
+        Public Sub DynamicInvocation_ControlFlowInReceiver()
+            Dim source = <![CDATA[
+Option Strict Off
+
+Class C
+    Private Sub M(c As Object, d1 As Object, d2 As Object)'BIND:"Private Sub M(c As Object, d1 As Object, d2 As Object)"
+        If(d1, d2).M2(c)
```

Comment:
>If(d1, d2).M2(c) [](start = 8, length = 16)

It doesn't look like this scenario is testing IDynamicInvocationOperation, consider adding ```call``` at the beginning of the line